packages/cli/package.json

```diff
@@ -20,6 +20,7 @@
     "@oclif/plugin-help": "^5.1",
     "chalk": "^2.4.2",
     "child-process-promise": "^2.2.1",
+    "copyfiles": "^2.3.0",
```

Comment:
This dependency is only used for the `compile` script. Could you move it to the `devDependencies` section?

Reply:
Moved to devDependencies ✅